packages/terra-folder-tree/translations/en-US.json

```diff
@@ -1,3 +1,7 @@
 {
+  "Terra.folder-tree.button.collapse-all": "Collapse All",
+  "Terra.folder-tree.button.expand-all": "Expand All",
+  "Terra.folder-tree.button.collapse-all-instructions": "Collapse all folders in the folder tree",
```

Comment:
nit: Can we make these consistent with en.json file having periods at the end of the sentence. Not sure how much it matters though.

Reply:
Good catch, those should be the same. Updated: [740723f](https://github.com/cerner/terra-framework/pull/1943/commits/740723f4b1fc77fb3bed0fe6de3dc1f69c408ec6)